templates/workspace_services/guacamole/terraform/main.tf

```diff
@@ -13,7 +13,10 @@ terraform {
 provider "azurerm" {
   features {
     key_vault {
-      purge_soft_delete_on_destroy = false
+      # Don't purge secrets on destroy (this would fail due to purge protection being enabled on keyvault)
+      purge_soft_deleted_secrets_on_destroy = false
+      # When recreating an environment, recover any previously soft deleted secrets
+      recover_soft_deleted_secrets = true
```

Comment:
Does the azurerm provider now handle secrets existing after recovery? My recollection was that when we had vaults recovered in the past, we had issues as the secrets needed to be imported

Reply:
Ugh this also shouldn't be there -- thanks for spotting, will revert this too